src/datastore/DistanceBetweenStadiums.csv

```diff
@@ -1,107 +1,54 @@
-58,Great American Ball Park,75,Tropicana Field,790
-58,Great American Ball Park,74,Sun Trust Park,375
-58,Great American Ball Park,76,Busch Stadium,310
-70,Guaranteed Rate Field,69,Comerica Park,240
-70,Guaranteed Rate Field,58,Great American Ball Park,250
-70,Guaranteed Rate Field,56,Wrigley Field,50
-60,Kauffman Stadium,76,Busch Stadium,235
-60,Kauffman Stadium,63,Globe Life Park in Arlington,460
-60,Kauffman Stadium,56,Wrigley Field,415
-60,Kauffman Stadium,62,Coors Field,560
-73,Marlins Park,74,Sun Trust Park,600
-73,Marlins Park,75,Tropicana Field,210
-73,Marlins Park,77,Nationals Park,930
-73,Marlins Park,65,Fenway Park,1255
-73,Marlins Park,57,Minute Maid Park,965
-79,Miller Park,71,Rogers Centre,430
-79,Miller Park,56,Wrigley Field,80
-79,Miller Park,59,Target Field,300
-57,Minute Maid Park,63,Globe Life Park in Arlington,230
-57,Minute Maid Park,75,Tropicana Field,790
-57,Minute Maid Park,73,Marlins Park,965
-57,Minute Maid Park,76,Busch Stadium,680
-57,Minute Maid Park,61,Chase Field,1115
-77,Nationals Park,68,Oriole Park at Camden Yards,50
-77,Nationals Park,78,PNC Park,195
-77,Nationals Park,74,Sun Trust Park,560
-77,Nationals Park,73,Marlins Park,930
-52,Oakland-Alameda County Coliseum,50,AT&T Park,50
-52,Oakland-Alameda County Coliseum,55,Dodger Stadium,340
-52,Oakland-Alameda County Coliseum,61,Chase Field,650
-68,Oriole Park at Camden Yards,77,Nationals Park,50
-68,Oriole Park at Camden Yards,67,Citizens Bank Park,90
-78,PNC Park,77,Nationals Park,195
-78,PNC Park,71,Rogers Centre,225
-78,PNC Park,72,Progressive Field,115
-78,PNC Park,58,Great American Ball Park,260
-78,PNC Park,66,Yankee Stadium,315
-54,Petco Park,62,Coors Field,830
-54,Petco Park,61,Chase Field,300
-54,Petco Park,53,Angel Stadium,110
-72,Progressive Field,78,PNC Park,115
-72,Progressive Field,69,Comerica Park,90
-72,Progressive Field,58,Great American Ball Park,225
-71,Rogers Centre,78,PNC Park,225
-71,Rogers Centre,79,Miller Park,430
-71,Rogers Centre,69,Comerica Park,210
-71,Rogers Centre,65,Fenway Park,430
-71,Rogers Centre,51,Safeco Field,2070
-51,Safeco Field,71,Rogers Centre,2070
-51,Safeco Field,59,Target Field,1390
-51,Safeco Field,50,AT&T Park,680
-74,Sun Trust Park,58,Great American Ball Park,375
-74,Sun Trust Park,77,Nationals Park,560
-74,Sun Trust Park,73,Marlins Park,600
-74,Sun Trust Park,63,Globe Life Park in Arlington,740
-59,Target Field,55,Dodger Stadium,1500
-59,Target Field,76,Busch Stadium,465
-59,Target Field,79,Miller Park,300
-59,Target Field,51,SafeCo Field,1390
-75,Tropicana Field,58,Great American Ball Park,790
-75,Tropicana Field,73,Marlins Park,210
-75,Tropicana Field,57,Minute Maid Park,790
-56,Wrigley Field,70,Guaranteed Rate Field,50
-56,Wrigley Field,79,Miller Park,80
-56,Wrigley Field,60,Kauffman Stadium,415
-66,Yankee Stadium,78,PNC Park,315
-66,Yankee Stadium,67,Citizens Bank Park,80
-66,Yankee Stadium,64,Citi Field,50
\ No newline at end of file
+# id, stadium name, id, stadium name, distance
```

Comment:
What happened in this file?

Reply:
Changed id's to reflect changes in MLBInfo file

Reply:
What happened to half of the file? It's like Thanos came in here and snapped his fingers lol

Reply:
I've discussed all of these changes with you already, I asked if you wanted me to remove duplicate edges that just went the other way

Reply:
But you don't address the changes in the PR description.